### PR TITLE
Add Let's Encrypt section 

### PR DESCRIPTION
**Pull request by staticsafe**
A short guide to Let's Encrypt certificate tool installation, certificate generation and automated renewal.

### CODE REVIEW

**Comment by M-D-M** [comment]
Could we make the cron script run weekly, instead of daily, since it reboots the nginx server?

**Comment by staticsafe** [comment]
@M-D-M it doesn't reboot the nginx server, only a reload.

This is what a reload does in the systemd unit file:
ExecReload=/usr/sbin/nginx -g 'daemon on; master_process on;' -s reload

**Comment by M-D-M** [comment]
Every day I learn something new! :+1: I feel like we could add one more line about "make sure to update your nginx config file," but I think most anybody would know to do that.

File: Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -335,6 +335,59 @@ WantedBy=multi-user.target
...
+We can do this with a cron job that runs daily:
+
+`nano /etc/cron.daily/letsencrypt-renew`
+
+Copy and paste this script into that file:
+
+```
+#!/usr/bin/env bash
+letsencrypt renew
+service nginx reload
```
Comment (Gargron) [inline Running-Mastodon/Production-guide.md]: `systemctl reload nginx`? Both ways of writing it work but Ubuntu 16.04 is systemd-land and we use systemctl in other parts of the documentation. Same comment for the cron restart lower down.

**Comment by staticsafe** [comment]
@Gargron I have amended it to use systemctl instead.

**Pull request merged**